docs/internal-documentation/maintenance/SupportCriteria.md

```diff
@@ -0,0 +1,124 @@
+Support criteria
+================
+This document hosts a set of guiding principles for selecting configuration
+parameter items to support.
+
+
+## General
+
+It is legitimate to avoid supporting items that are otherwise perfectly eligible
+for support, just for the sake of limiting the total number of configuration
+tuples.
+
+
+## Architectures
+
+Can we say something about how we chose to support only amd64/x86_64 and nothing
+else?
+
+
+## Operating systems
+
+Can we say something about how the current quartet of operating systems was
+chosen?
```

Comment:
I think we should only state that we only actively test Zonemaster under the selected OSs. I do not think that we have to motivate the choice.